doc/modules/cassandra/pages/new/configuration.adoc

```diff
@@ -194,3 +200,6 @@ hinted_handoff_enabled: false
 ....
 
 https://issues.apache.org/jira/browse/CASSANDRA-17379[CASSANDRA-17379] was opened to improve the user experience and deprecate the overloading.
+By default, we refuse starting Cassandra with a config containing both old and new config keys for the same parameter. Start
+Cassandra with `-Dcassandra.allow_new_old_config_keys=true` to override. For historical reasons duplicate config keys
+in `cassandra.yaml` are allowed by default, start Cassandra with `-Dcassandra.allow_duplicate_config_keys=false` to disallow this.
```

Comment:
I added this as the commit I provided in CASSANDRA-17379 for the doc to be updated was not committed